azero/scripts/2_l2_setup.ts

```diff
@@ -40,20 +40,20 @@ async function main(): Promise<void> {
   console.log(`wAZERO address set to ${wAzeroAddress}`);
   await most.tx.setWazero(wAzeroAddress);
 
-  if (config.dev) {
-    console.log("Creating WrappedAzero instance");
-    const wAzero = new WrappedAzero(wAzeroAddress, deployer, api);
-    const deposit = BigInt(10000 * 10 ** 12);
+  console.log("Creating WrappedAzero instance");
+  const wAzero = new WrappedAzero(wAzeroAddress, deployer, api);
+  const deposit = BigInt(10000 * 10 ** 12);
```

Comment:
mv 10k to config

Reply:
Done. I also added example `l2_testnet.json.example` file